docs/cli-reference/start.md

```diff
@@ -77,9 +77,10 @@ Most of the options can be set via environment variables. The environment variab
 
 Below are additional environment variables that can be set:
 
-| <div style="width:360px">Flag</div> | Description                                              |
-| ----------------------------------- | -------------------------------------------------------- |
-| `HF_ENDPOINT`                       | Hugging Face Hub endpoint. e.g., `https://hf-mirror.com` |
+| <div style="width:360px">Flag</div>        | Description                                              |
+| -----------------------------------        | -------------------------------------------------------- |
+| `HF_ENDPOINT`                              | Hugging Face Hub endpoint. e.g., `https://hf-mirror.com` |
+| `GPUSTACK_DISABLE_DYNAMIC_LINK_LLAMA_BOX`  | Dynamic links are used by default,this environment variable can disable dynamic linking behavior to enable static linking. e.g., `true`           |
```

Comment:
Dynamic linking is used by default. Setting this to `true` enables static linking.